test/relations.test.js

```diff
@@ -222,16 +238,21 @@ describe('relations', function() {
             should.not.exist(err);
             should.exist(ch);
             ch.should.have.lengthOf(3);
-
             var chapters = book.chapters();
             chapters.should.eql(ch);
-
-            book.chapters.getAsync({order: 'name DESC'}, function(e, c) {
+            var nameFilter = isCassandraConnector ? {} : {order: 'name DESC'};
+            book.chapters.getAsync(nameFilter, function(e, c) {
```

Comment:
Remove the filter completely and use assertions that don't assume any order of the returned books.